Change ScrollPane in table (#47)
se-edu/guides: tutorials/javaFxPart2.md

![Label](images/javafx/MockupLabel.png) | Label
![Button](images/javafx/MockupButton.png) | Button
![TextField](images/javafx/MockupTextField.png) | TextField
<img src="images/javafx/MockupScrollPane.png" width="300" /> | ScrollPane | ScrollPane

> {{ icon_tip }} You may be wondering why a `ScrollPane` is a `Control` and not a `Pane`, as its name suggests. `ScrollPane` extends the `Control` class in JavaFX because it represents a UI "widget" that the user directly interacts with (such as `Label`, `Button`, and `TextField`), whereas a `Pane` is a mere container that holds other components. Of course, a `ScrollPane` has the functionality of a container, allowing it to show a <tooltip content="gives a rectanular view into a larger content which exceeds the available screen/stage area">viewport</tooltip> of its content. `Scrollpane` contains one node, whereas a `Pane` can contain multiple nodes.

Author: yes... thank you so much.

![TextField](images/javafx/MockupTextField.png) | TextField

> ![ScrollPane](images/javafx/MockupScrollPane.png) | ScrollPane

Reviewer: ```suggestion
<img src="images/javafx/MockupScrollPane.png" width="???" /> | ScrollPane
```

@laney0808 You can use this syntax to control the size of the image.

> :bulb: You may be wondering why a ScrollPane is a `Control`, and not a `Pane`, as its name suggests. `ScrollPane` extends the `Control` class in JavaFX, because it represents a UI "widget" which the user directly interacts with (such as `Label`, `Button`, and `TextField`), whereas a `Pane` is a mere container that contains the other components. Of course, ScrollPane has the functionality of a container, allowing it to contain, or rather show a viewport of, its content.

Reviewer: ```suggestion
{{ icon_tip }} You may be wondering why a ScrollPane is a `Control`, and not a `Pane`, as its name suggests. `ScrollPane` extends the `Control` class in JavaFX, because it represents a UI "widget" which the user directly interacts with (such as `Label`, `Button`, and `TextField`), whereas a `Pane` is a mere container that contains the other components. Of course, ScrollPane has the functionality of a container, allowing it to contain, or rather show a viewport of, its content.
```

For better consistency.

Author: Sure! Thank you prof. A new commit is made.

> 

Reviewer: @laney0808 Do readers know what a viewport is, by this point? If not, you can use a tooltip to give a definition.
```
<tooltip content="i.e., ...">viewport</tooltip>
```

Author: Sure!

Reviewer: The term `window` has different meanings, which might confuse the reader.

> {{ icon_tip }} You may be wondering why a ScrollPane is a `Control`, and not a `Pane`, as its name suggests. `ScrollPane` extends the `Control` class in JavaFX, because it represents a UI "widget" which the user directly interacts with (such as `Label`, `Button`, and `TextField`), whereas a `Pane` is a mere container that contains the other components. Of course, ScrollPane has the functionality of a container, allowing it to contain, or rather show a <tooltip content="gives a window into a larger view when the content exceeds the available screen/stage area">viewport</tooltip> of, its content.

Reviewer 2: Added some suggestions on consistent inline codes and grammatical/typo suggestions!

```suggestion
{{ icon_tip }} You may be wondering why a `ScrollPane` is a `Control` and not a `Pane`, as its name suggests. `ScrollPane` extends the `Control` class in JavaFX because it represents a UI "widget" that the user directly interacts with (such as `Label`, `Button`, and `TextField`), whereas a `Pane` is a mere container that holds other components. Of course, `ScrollPane` has the functionality of a container, allowing it to show a <tooltip content="gives a window into a larger view when the content exceeds the available screen/stage area">viewport</tooltip> of its content.
```
